index.js

```diff
@@ -1,39 +1,26 @@
 'use strict';
-var cheerio = require('cheerio');
-var got = require('got');
 var Promise = require('pinkie-promise');
+var utils = require('./lib/utils');
+var stats = require('./data.json').stats;
```

Comment:
Should I rename this to `cachedStats` or something? Because below I use `stats` as well. This might be confusing.